src/components/llama-select-fab.test.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+const path = require('path');
+
+// Building the website and starting the server takes ~7 seconds on a MacBook Pro.
+jest.setTimeout(15000);
+
+describe('LlamaSelectFab', () => {
+  it('fires an event when a file has been selected', async () => {
+    await page.goto('http://localhost:8888', { 'waitUntil': 'domcontentloaded' });
+
+    const selectHandle = await page.$('llama-select-fab');
+    const inputHandle = await selectHandle?.$('input[type="file"]');
+    expect(inputHandle).not.toBeNull();
```

Comment:
frameworks that try to pretend they're just normal English sentences are a pet peeve of mine